dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassPathEntryTest.java

```diff
@@ -143,6 +144,7 @@ private static Artifact resolveArtifact(String coordinates) throws ArtifactResol
 
     Artifact artifact = new DefaultArtifact(coordinates);    
     ArtifactRequest artifactRequest = new ArtifactRequest();
+    artifactRequest.addRepository(CENTRAL);
```

Comment:
why is this change needed? I'm surprised upgrading dependencies caused this.

Reply:
It has been broken in that every time we upgrade truth dependency, we had to upgrade this test case. For example: https://github.com/GoogleCloudPlatform/cloud-opensource-java/pull/1809#issuecomment-727011693 (November last year)
It's because `system.resolveArtifact` was not downloading the Truth artifact and the test case has been relying on local Maven repository to have the artifact.